whylog/tests/tests_config/test_files/parsers.yaml

```diff
@@ -5,17 +5,20 @@ name: lostdata
 primary_key_groups: [1]
 regex_str: '^(\d\d\d\d-\d\d-\d\d \d\d:\d\d:\d\d) Data is missing at (.*)\. Loss = (.*)
   GB\. Host name: (.*)$'
+line_content: "2016-04-12 23:54:43 Data is missing at comp2. Loss = 150 GB. Host name: host2"
```

Comment:
This filed was added by Ewa. It will be use for display our parser in vim plugin